freeipa/src/main/resources/freeipa-salt/salt/ccm/scripts/socket-wait-cleanup.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+: "${SOCKET_WAIT_LIMIT:=1000}"
+socket_wait_count=$(netstat -nat | egrep 'CLOSE_WAIT' | wc -l)
```

Comment:
what if an another service will do this (does not close connections properly), is this will restart CCM service?
can we check that as well these sockets are all CCM related in the script (you should be able to match process id against the open connections)?

Reply:
PR was updated, please check. Relevant part:
socket_wait_count=$(netstat -nat -p | grep ':9443\|:443' | grep CLOSE_WAIT | wc -l)